shell/platform/darwin/ios/rendering_api_selection.mm

```diff
@@ -15,40 +15,17 @@
 
 namespace flutter {
 
-bool ShouldUseSoftwareRenderer() {
-  return [[[NSProcessInfo processInfo] arguments] containsObject:@"--force-software"];
-}
-
```

Comment:
Why are we dropping this?

Reply:
This was only added recently so I could check all three variants in one build. There is no reason to support software rendering on iOS.

Reply:
So it's set somewhere else for simulator?

Reply:
Roger https://github.com/flutter/engine/pull/17431/files#diff-68cfd589fe3ae373eb867fda891a0c40L56